static/app/components/ai/SeerIcon.tsx

```diff
@@ -55,7 +54,7 @@ const InteractionWrapper = styled('div')`
   }
 `;
 
-const SeerIcon = forwardRef<SVGSVGElement, SVGIconProps>((props, ref) => {
+function SeerIcon({ref, ...props}: SVGIconProps) {
   return (
     <SvgIcon {...props} ref={ref} viewBox="0 0 30 30">
```

Comment:
we can actually remove the props spreading and `ref` variable destruction because we could just forward the prop like all other props:

```
function SeerIcon(props: SVGIconProps) {
  return (
    <SvgIcon {...props} viewBox="0 0 30 30">
```

I plan to do this in a follow-up because it creates even larger diffs and is irrelevant to the actual forwardRef removal